include/triggeralgs/Logging.hpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Logging.hpp Common logging declarations in triggeralgs
+ *
+ * This is part of the DUNE DAQ , copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+#ifndef TRIGGERALGS_INCLUDE_LOGGING_HPP_
+#define TRIGGERALGS_INCLUDE_LOGGING_HPP_
+
+namespace dunedaq {
+namespace triggeralgs {
+namespace logging {
+
+/**
+* @brief Common name used by TRACE TLOG calls from this package
+*/
+enum
+{
+  TLVL_VERY_IMPORTANT = 1,
+  TLVL_IMPORTANT      = 2,
+  TLVL_GENERAL        = 3,
+  TLVL_DEBUG_INFO     = 4,
+  TLVL_DEBUG_LOW      = 5,
+  TLVL_DEBUG_MEDIUM   = 10,
+  TLVL_DEBUG_HIGH     = 15,
+  TLVL_DEBUG_ALL      = 20
```

Comment:
`IMPORTANT` and `VERY_IMPORTANT` is strange naming (I guess `GENERAL` and `DEBUG_INFO` are too). Is the naming similar across the rest of the DAQ? The usual naming for different logging verbosity in the industry is:
* `TRACE` for everything, so `TLVL_DEBUG_ALL`
* `DEBUG` for the debugging messages (here it makes sense to have low, medium, high)
* `INFO` would be your `VERY_IMPORTANT`, `IMPORTANT` and/or `DEBUG_INFO`,
* `WARN` for warnings (we have different system for that),
* `ERROR` for errors (we have different system for that),
* `FATAL` for fatal errors that should crash (we have different system for that).

What are the namings in the rest of the DAQ?

Reply:
This naming is completely custom. As far as I know, there is no consistency across dunedaq. There are some repos that have a version of this, example [in-file enum here](https://github.com/DUNE-DAQ/flxlibs/blob/production/v4/src/CardWrapper.cpp#L26-L31) and [separare logging header here](https://github.com/DUNE-DAQ/readoutlibs/blob/production/v4/include/readoutlibs/ReadoutLogging.hpp#L18-L29). Other places simply use integers [example here](https://github.com/DUNE-DAQ/hsilibs/blob/production/v4/plugins/HSIReadout.cpp#L250), and these do go to high values ([example here](https://github.com/DUNE-DAQ/iomanager/blob/production/v4/src/network/ConfigClient.cpp#L89)).

I just made that list according to what made sense to me, and wanted it to (a) be granural at the lower levels and (b) roughly cover the available space for levels. Open to changes. As you mentioned, we are not using standards from the industry (or at least not closely).